### PR TITLE
Nicolas' proposal for ergonomic fixes

### PR DESCRIPTION
**Pull request by fvanderbiest**
https://jira.camptocamp.com/browse/GSKAN-232

### CODE REVIEW

File: ckanext/pigma_theme/plugin.py [hunk 1]
```diff
@@ -28,6 +28,12 @@ class Pigma_ThemePlugin(plugins.SingletonPlugin):
     plugins.implements(plugins.ITranslation)
     plugins.implements(ISpatialHarvester, inherit=True)
     plugins.implements(plugins.ITemplateHelpers)
+    plugins.implements(plugins.IRoutes, inherit=True)
+
+    # IRoutes
+    def before_map(self, m):
+        m.redirect('/', '/dataset')
```
Comment (fvanderbiest) [inline ckanext/pigma_theme/plugin.py:35]: I'm not sure this is expected

**Comment by fvanderbiest** [comment]
For the record, `organization_item.html` diff is
```diff
@@ -13,7 +13,7 @@
           <h3 class="media-heading"><a href={{ url }}>{{ organization.title or organization.name }}</a></h3>
         {% endblock %}
         {% block organization_item_header_description %}
-          {% if organization.description %}
+          {% if false && organization.description %}
             {% if truncate == 0 %}
               <p>{{ h.markdown_extract(organization.description)|urlize }}</p>
             {% else %}
@@ -24,7 +24,7 @@
       </header>
     {% endblock %}
     {% block organization_item_content %}
-      {% set list_class = "list-unstyled dataset-list" %}
+      {% set list_class = "unstyled dataset-list" %}
       {% set item_class = "dataset-item module-content" %}
       {% snippet 'snippets/package_list.html', packages=organization.packages, list_class=list_class, item_class=item_class, truncate=120 %}
     {% endblock %}
```

**Comment by fvanderbiest** [comment]
@tonio can you please check / merge this PR this morning ? Thanks.

**Review by tonio — APPROVED** [review]
Not complete compared to passed doc, but good enough to be merged.

**Pull request merged**